.gitignore

```diff
@@ -3,6 +3,7 @@ __pycache__/
 *.py[cod]
 *$py.class
 
+**.idea/**
```

Comment:
are the double `**` necessary here? i would have thought one either side would be sufficient